modules/gateway/rpc.go

```diff
@@ -114,6 +114,7 @@ func (g *Gateway) threadedHandleConn(conn modules.PeerConn) {
 		g.log.Printf("WARN: incoming conn %v requested unknown RPC \"%v\"", conn.RemoteAddr(), id)
 		return
 	}
+	g.log.Printf("INFO: incoming conn %v requested RPC \"%v\"", conn.RemoteAddr(), id)
```

Comment:
*if you really want the log, have it write the log in debug mode. bitcoind will make stupid-large logs (sometimes running into the hundreds of GBs) if you run it in debug/verbose mode. It's probably okay for us to take a similar approach. Non-debug logs only log when something unanticipated goes wrong (so, failed RPCs and random network drops would not merit logs), and debug logs just dump data.